docs/source/pillars/computing_technology.md

```diff
@@ -112,6 +112,15 @@ _The ability to run analyses requiring more compute resources, or more specialis
     For example, when using physical compute resources all sensitive data must be securely wiped before another user is given access to that same node.
     In a cloud hosted TRE virtual machines should be destroyed and recreated.
   - Mandatory
+* - TREs working with big datasets could integrate with large-scale data analytics tools.
+  - For example Spark and Hadoop can be used for distributed computing across a cluster.
+    Where data included in TREs are impractically large for single-machine computing, this will be an advantage.
+  - Optional
+* - TREs integrating with cloud-native managed services should consider the associated risks.
+  - Public clouds provide many different managed services which might include Jupyter notebooks, big data pipelines, databases, API management or serverless computing.
+    Examples include Databricks for automated cluster management and cloud machine-learning platforms such as SageMaker (AWS) or Azure Machine Learning.
+    Although the cloud provider is responsible for providing these services, the TRE operator must ensure that using them does not compromise the security of the TRE.
+  - Optional
```

Comment:
I might have missed the conversation around this.

It is definitely important, although I feel a bit like it goes without saying. Is there anything about these services _in particular_ that means we need to consider the risks? For example, would it be any different than using your cloud providers LDAP/AD service?

Reply:
Example: you use a managed service such as SageMaker which provides access via a unique URL for that instance. Since it's a TRE you must ensure the networking is customised to only allow access to that URL from a network inside the TRE.

Reply:
Expanding further, managed services hide a lot of the complexity so it would be quite easy to make a service available inside the TRE without realising what it actually allows. For something like LDAP/AD it's not as big a problem because the purpose of that service is bounded, but for anything supporting interactive user access there's a danger of it leading to an opening.

Reply:
Thanks @manics. I think I can see the distinction. A lot of the cloud-native services see 'access from anywhere' as a feature, where as other software requires you to be proactive to route connections.

I still think it is a bit obvious, but I don't object enough to suggest changing or removing the statement.